src/Image/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { CSSProperties } from 'react';
+
+interface ImgProps {
+    alt: string;
+    height?: number;
+    src: string;
+    title: string;
+    width?: number;
+}
+
+interface AmpImgProps extends ImgProps {
+    layout: string;
+}
+
+// TODO: Get rid of the anys
+interface ImageProps extends ImgProps {
+    globalSettings: any;
+    utils: any;
+}
+
+const responsiveImgStyles: CSSProperties = {
+    width: '100%'
+};
+
+const AmpImg = (props: AmpImgProps) => <amp-img {...props} />;
+const Img = (props: ImgProps) => (
+    <picture>
+        <img
+            {...props}
+            style={props.height ? undefined : responsiveImgStyles}
+        />
+    </picture>
+);
+
+const Image = ({ alt, height, src, title, utils, width }: ImageProps) => {
+    const imgProps = { alt, height, src, title, width };
```

Comment:
You could condense this a little by using rest syntax:
```suggestion
const Image = ({ utils, ...imgProps }: ImageProps) => {
```

Reply:
Yeah I originally had that, but `globalSettings` was also getting passed to the img/amp-img tags.